### DISCUSSION TITLE
part-ii/2019-87/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#235] [Question comments]
Part II 2019 Logic and Set Theory: Paper 3, Section II, I 

https://questions.tripos.org/part-ii/2019-87/

**Comment by artie2000** [comment]
The formula $\text{rk}x=\sup_{y\in x}\left\{(\text{rk}y)^+\right\}$ is your friend.

<details>
<summary>(i):</summary>

True because $\max_{1\leq i\leq k}\left\{(\text{rk}a_i)^+\right\}=\left(\max_{1\leq i\leq k}\{\text{rk}a_i\}\right)^+$.

</details>
<details>
<summary>(ii):</summary>

False: consider $V_{\omega+1}$.

</details>
<details>
<summary>(iii):</summary>

True. Try to prove the contrapositive.

</details>
<details>
<summary>(iv):</summary>

True: $\text{rk}\alpha=\alpha$!

</details>

**Reply by dropdbassett** [reply]
Something that might be a little slicker/more direct for (iii):
<details><summary>(iii)</summary>

$\mathrm{rk}(x) = \sup\{\mathrm{rk}(\{y\}) : y \in x\}$. Since $\{y\} \subseteq x$ is finite, it has rank at most $\alpha$, and so the same is true for the supremum.
</details>
